### PR TITLE
Fix XDR close reason

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-14732

## Description
Because of mapping the `status` field for common types in XDR - the status in XDR was incorrectly change to wrong value 
in case the XSOAR incident was automatically closed by the system

after discussion with @ssokolovich 
we decided to remove the problematic mapping and fix something in the code

the build will fail on the remove of the `status` in the mapper @yaakovi FYI

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

### CODE REVIEW

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py [hunk 1]
```diff
@@ -2571,24 +2579,29 @@ def handle_user_unassignment(update_args):
...
-    if inc_status == 2:
-        if MIRROR_IN_CLOSE_REASON not in update_args.get('closeNotes', ''):
-            update_args['resolve_comment'] = update_args.get('closeNotes', '')
-            update_args['status'] = XSOAR_RESOLVED_STATUS.get(update_args.get('closeReason', 'Other'))
-            demisto.debug(f"Closing Remote incident with status {update_args['status']}")
-        else:
-            demisto.debug('Ignore closing Remote incident because incident closed during mirroring in')
+def handle_outgoing_issue_closure(remote_args):
+    update_args = remote_args.delta
+    current_remote_status = remote_args.data.get('status')
```
Comment (tkatzir) [inline Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:2584]:
```suggestion
    current_remote_status = remote_args.data.get('status') if remote_args.data else None
```
Check that `remote_args.data` isn't None.

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py [hunk 2]
```diff
@@ -2571,24 +2579,29 @@ def handle_user_unassignment(update_args):
...
+    # force closing remote incident only if:
+    #   XSOAR incident are closed
+    #   the closingUserId was changed
```
Comment (tkatzir) [inline Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:2587]:
```suggestion
    #   and the closingUserId was changed
```

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py [hunk 3]
```diff
@@ -2571,24 +2579,29 @@ def handle_user_unassignment(update_args):
...
+    #   remote incident aren't already closed
```
Comment (tkatzir) [inline Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:2588]:
```suggestion
    #   and the remote incident isn't already closed
```

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py [hunk 4]
```diff
@@ -2571,24 +2579,29 @@ def handle_user_unassignment(update_args):
...
-            update_args['resolve_comment'] = update_args.get('closeNotes', '')
-            update_args['status'] = XSOAR_RESOLVED_STATUS.get(update_args.get('closeReason', 'Other'))
-            demisto.debug(f"Closing Remote incident with status {update_args['status']}")
-        else:
-            demisto.debug('Ignore closing Remote incident because incident closed during mirroring in')
+def handle_outgoing_issue_closure(remote_args):
+    update_args = remote_args.delta
+    current_remote_status = remote_args.data.get('status')
+    # force closing remote incident only if:
+    #   XSOAR incident are closed
```
Comment (tkatzir) [inline Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:2586]:
```suggestion
    #   The XSOAR incident is closed
```

File: Packs/Core/ReleaseNotes/1_3_13.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Investigation & Response
+- Fixed an issue where *Close Reason* was incorectly changed by mirroring out.
```
Comment (tkatzir) [inline Packs/Core/ReleaseNotes/1_3_13.md:4]:
```suggestion
- Fixed an issue where *Close Reason* was incorrectly changed by mirroring out.
```

File: Packs/CortexXDR/ReleaseNotes/4_8_22.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Fixed an issue where *Close Reason* was incorectly changed by mirroring out.
```
Comment (tkatzir) [inline Packs/CortexXDR/ReleaseNotes/4_8_22.md:4]:
```suggestion
- Fixed an issue where *Close Reason* was incorrectly changed by mirroring out.
```

**Review by tkatzir — APPROVED** [review]
Looks great! Approved, but please fix my minor comments before merging.

**Comment by ilappe** [comment]
@tkatzir need force merge 
as the validate fail on [remove status from mapping](https://app.circleci.com/pipelines/github/demisto/content/200856/workflows/fc49e9bf-86c5-452b-8e85-4dc520775736/jobs/586944)

**Pull request merged**